### PR TITLE
Updates to node_exporter start cmd and grok patterns options

### PR DESCRIPTION
**Pull request by r2ronoha**
userdata:
- collector.filesystem.ignored-mount-points ([to eliminate errors](https://github.com/prometheus/node_exporter/issues/773))
- grok patterns
- dpkg cleanup (userdata fails due to old lock files and packages preventing docker installation)

### CODE REVIEW

File: aws/templates/user-data.sh [hunk 1]
```diff
@@ -120,6 +130,7 @@ ExecReload=/bin/kill -HUP $MAINPID
 ExecStart=/usr/local/bin/node_exporter $COLLECTORS \
     --web.listen-address=:$NODE_EXPORTER_PORT \
     --web.telemetry-path="$NODE_EXPORTER_PATH"
+    --collector.filesystem.ignored-mount-points="^/(sys|proc|dev|rootfs/var/lib/docker|rootfs/run/docker/netns)(\$|/)"
```
Comment (r2ronoha) [inline aws/templates/user-data.sh]: As advised per https://github.com/prometheus/node_exporter/issues/773

File: aws/templates/user-data.sh [hunk 2]
```diff
@@ -55,6 +55,16 @@ filesystem_setup() {
   echo `blkid $DEVICE -o export | grep -E "UUID=+*"` $RANCHER_DIR ext4 defaults,nofail 0 2 >> /etc/fstab
 }
 
+clean_packages() {
```
Comment (r2ronoha) [inline aws/templates/user-data.sh]: Should we have our own image as for the rest of K8s nodes?

**Review by coufalja — DISMISSED** [review]

**Comment by coufalja** [comment]
Please rebase

**Comment by r2ronoha** [comment]
replaced by https://github.com/WanderaOrg/terraform-rancher/pull/16

**Pull request closed without merging**